kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -213,6 +212,14 @@
     height: 35px
```

Comment:
I think this might be a better note for the people making the mock ups. @whitzhu @66eli77  ?